src/main/java/software/amazon/awssdk/crt/CrtPlatform.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.awssdk.crt;
+
+import software.amazon.awssdk.crt.utils.PackageInfo;
+
+public abstract class CrtPlatform {
```

Comment:
Each platform is free to include files which implement this and override test environment and packaging. Android, in particular, does everything DIFFERENT, so we need a shim somewhere.